aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -156,6 +156,21 @@ Tensor& addr_out(Tensor &result, const Tensor& self, const Tensor& vec1, const T
   return at::_addr_out(result, self, vec1, vec2, beta, alpha);
 }
 
+Tensor& ger_out(Tensor &result, const Tensor& self, const Tensor& vec2) {
+  check_1d(self, "self", "ger");
+  check_1d(vec2, "vec2", "ger");
+  if (result.dim() != 2 || result.size(0) != self.size(0) || result.size(1) != vec2.size(0)) {
```

Comment:
nit: I think it's clearer if you do something like
```
if (result.sizes() != {self.size(0), vec2.size(0)}) {
   result.resize_({...});
}
```

Reply:
Will do if there are more dimensions to check. For 2 dimensions, I think this is still fine and save a memory allocation, faster.